packages/repository/src/__tests__/fixtures/repositories/note.repository.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Constructor} from '@loopback/core';
+import {DataSource} from 'loopback-datasource-juggler';
```

Comment:
We should avoid direct import to `loopback-datasource-juggler`.  Use `juggler.DataSource` instead.

```
import {juggler, DefaultCrudRepository} from '../../..';
```